base/sharedarray.jl

```diff
@@ -147,8 +147,8 @@ SharedArray{T}(m::Integer, n::Integer; kwargs...) where {T} =
 SharedArray{T}(m::Integer, n::Integer, o::Integer; kwargs...) where {T} =
     SharedArray{T,3}(m, n, o; kwargs...)
 
-function (::Type{SharedArray{T,N}})(filename::AbstractString, dims::NTuple{N,Int}, offset::Integer=0;
-                                    mode=nothing, init=false, pids::Vector{Int}=Int[]) where T where N
+function SharedArray{T,N}(filename::AbstractString, dims::NTuple{N,Int}, offset::Integer=0;
+                          mode=nothing, init=false, pids::Vector{Int}=Int[]) where T where N
```

Comment:
Yes they can, but the rest of the file has chosen to have them as `where T where N` so for consistency I have it like this